.python-version

```diff
@@ -0,0 +1 @@
+3.10.4
```

Comment:
Does this need to be committed?

Reply:
For those who are using pyenv, yes.

I have to develop for 3.8 in offers, 3.9 when I use Cloud Functions or App Functions and 3.10 for playbook and this repo.

Reply:
What will be keeping this up to date, how will we be actually checking compatibility here?

Reply:
But this repo's python version range is set in poetry already.

Reply:
> But this repo's python version range is set in poetry already, are you not using that?

Is there another tool to manage python version like terraform-switcher that can pick python version from pyproject.toml?

Or you just have latest python installed on your laptop? I can not just use latest python from brew or apt, for offers 3.10 will not work.

But btw, nevermind we just switch to containers and utilize multi stage,.

Reply:
The thing is just, if nothing uses this except you, then it will just become out of date, because nothing updates it, nothing uses it. If you want to use pyenv in the CI then you should set that up.

Reply:
> Or you just have latest python installed on your laptop? I can not just use latest python from brew or apt, for offers 3.10 will not work.

I have a bunch of pythons installed, and the right one is selected by poetry, if you cant use 3.10 for offers then poetry for offers should say python version should be below 3.10.